vllm/model_executor/layers/quantization/utils/w8a8_utils.py

```diff
@@ -316,7 +316,9 @@ def __init__(self,
         # as it breaks with dynamic shapes.
         if pad_output is None:
             config = get_current_vllm_config().compilation_config
-            pad_output = config.level < CompilationLevel.PIECEWISE and \
+            # TODO: temporary solution, see: https://github.com/vllm-project/vllm/pull/22360
+            level = config.level or CompilationLevel.NO_COMPILATION
```

Comment:
I agree this is bad, but it does bring the old behavior back...
In the old behavior if you set config.level manually, then it gets picked up here, and then finally it will get overridden to 3. Otherwise config.level defaults to 0, gets picked up here, and finally gets overridden to 3. That's exactly what this PR is doing